### PR TITLE
fix text.dart documentation

### PR DESCRIPTION
**Pull request by jsh-me**
## Description
I modified TextAlign docs because the descriptions of `TextAlign.center` and `Center Widget` are so similar that they cause confusion.

## Related Issues

https://github.com/flutter/flutter/issues/5805

## Checklist

Before you create this PR, confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter/lib/src/widgets/text.dart [hunk 1]
```diff
@@ -120,7 +120,7 @@ class DefaultTextStyle extends InheritedTheme {
   /// The text style to apply.
   final TextStyle style;
 
-  /// How the text should be aligned horizontally.
+  /// How every line of the text in Text widget should be aligned horizontally.
```
Comment (gspencergoog) [inline packages/flutter/lib/src/widgets/text.dart:123]:
```suggestion
  /// How each line of text in the Text widget should be aligned horizontally.
```

**Review by gspencergoog — CHANGES_REQUESTED** [review]
Thanks for the PR!  Yes, this needs some clarification.

I have a suggestion for better wording, though.

**Comment by jsh-me** [comment]
@gspencergoog 
Thank you! I fixed it. Could you check it again?

**Review by gspencergoog — APPROVED** [review]
32384589-a60f0e74-c078-11e7-9bc1-e5b5287aea9d

I'll submit this once the tree is green.

**Pull request merged**